.github/workflows/nightly-conda.yml

```diff
@@ -0,0 +1,120 @@
+name: TorchArrow Nightly Build and Test for Conda
+on:
+#  schedule:
+#    - cron: "0 0 * * *"
+  pull_request:
+    branches:
+      - main
+      # For PR created by ghstack
+      - gh/*/*/base
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  workflow_call:
+    secrets:
+      AWS_PYTORCH_UPLOADER_ACCESS_KEY_ID:
+        required: true
+      AWS_PYTORCH_UPLOADER_SECRET_ACCESS_KEY:
+        required: true
```

Comment:
You don't need this part. `workflow_call` is used to invoke this workflow script from another workflow.